packages/composite-checkout/README.md

```diff
@@ -0,0 +1,559 @@
+
+### usePaymentMethodData()
+
+A React Hook that will return a two element array. The first element is an object representing the current state stored for the currently selected payment method (or null if none is selected). The second element is a function that will replace the state of the currently selected payment method (the function will be null if no payment method is selected).
+
+### usePaymentMethodId()
+
+A React Hook that will return a two element array. The first element is a string representing the currently selected payment method (or null if none is selected). The second element is a function that will replace the currently selected payment method.
+
+## 💰 FAQ
+
+### How do I use Credits and Coupons?
+
+Credits, coupons, and discounts are all ways that the line items and the total can be modified, so they must be handled by the parent component.
+
+### Do line items need to be products?
+
+No, line items can be anything. The most common use is for products, taxes, subtotals, discounts, and other adjustments to the total price. However, line items can also contain other information that you'd like to display in the review step. If you customize the Order Review step, you will also be able to decide how each line item is presented. Just be aware that line items may be passed along to the server when the purchase is made, depending on the payment method. Check the documentation for each payment method to determine if there are any specific requirements there.
+
+### Do line items amount properties have to have an integer value?
+
+The primary properties used in a line item by default are `id` (which must be unique), `label` (with an optional `subLabel`), and `amount.displayValue`. The other properties (`type`, `amount.currency`, `amount.value`) are not used outside custom implementations, but it's highly recommended that you provide them. As requirements and customizations change, it can be helpful to have a way to perform calculations, conversions, and sorting on line items, which will require those fields. If any required field is undefined, an error will be thrown to help notice these errors as soon as possible.
+
+### Can I add custom properties to line items?
+
+To maintain the integrity of the line item schema, adding custom fields is discouraged, but allowed. If you need specific custom data as part of a line item so that it can be used in another part of the form, it's recommended to pass the line item object through a helper function to collect the extra data rather than serializing it into the line item itself. This will make that data collection more testable. However, if the data collection process is expensive or slow, and caching isn't an option, it may make sense to preload the data into the line items.
+
+## 💰 Development
+
+In the root of the monorepo, run `npm run composite-checkout-demo` which will start a local webserver that will display the component.
```

Comment:
This is fine — incorporating to devdocs where rest of the stuff in calypso is showcased/developed would feel more aligned but not a Biggie. Prolly faster this way?

Reply:
Yes. I did consider this, but devdocs is quite slow and loads a lot of unnecessary things (ie: calypso) for this project. The best solution would be Storybook (#33166) but this is our temporary solution until we have that. We could also put it in devdocs once we are not in active development.